Helper/Data.php

```diff
@@ -61,6 +57,13 @@ public function getApiEndpoint()
         );
     }
 
+    public function getActivityEndpoint()
+    {
+        return $this->config->getValue(
```

Comment:
OK, do you remember that we discussed that there should be a default.
- If there is config value -> use the config value
- If there is no config values -> "https://p.metrilo.com"

Same thing we did for the API endpoint (the default there being "https://tracking.metrilo.com")

Reply:
Yes, I do remember when this default value was implemented in ApiClient class but it was never set to "https://tracking.metrilo.com" but to "https://p.metrilo.com" instead. In my local db the metrilo_analytics/general/api_endpoint path value is set to "https://tracking.metrilo.com". This brings the question about should I edit the development/installation file and remove the config:set options for api_endpoint and the newly added activity_endpoint (since they are required only for QA env) and refactor the ApiClient class $apiEndpoint variable to default to the correct tracking url and implement the same logic in activity helper where $activityEndpoint will default to activity url.